cmake/dependencies/ABLASTR.cmake

```
@@ -102,7 +102,7 @@ set(ImpactX_ablastr_src ""
 set(ImpactX_ablastr_repo "https://github.com/ECP-WarpX/WarpX.git"
     CACHE STRING
     "Repository URI to pull and build ABLASTR from if(ImpactX_ablastr_internal)")
-set(ImpactX_ablastr_branch "278f3cd3db8889219e468699a8475d664ac5f2d3"
+set(ImpactX_ablastr_branch "8b8760d2cb9d288b3c6f3f52c2bf2536f232afc0"
```

fix https://github.com/ECP-WarpX/WarpX/pull/2953
```suggestion
set(ImpactX_ablastr_branch "945568b98c87f89ce9199bfd72e6660058e614fc"
```